applications/hybrid.md

```diff
@@ -0,0 +1,234 @@
+# Hybrid Block Explorer
+
+- **Team Name:** Jonathan Brown
+- **Payment Address:** 0x36a7401F269555916a0243E586234D3Bbf5A0c36 (DAI)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+
+## Project Overview :page_facing_up:
+
+This application is in response to two RFPs:
+- Multi-chain Block Explorer https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Under%20Development/multi-chain-block-explorer.md
+- ink! smart contract block explorer. https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Under%20Development/ink_smart_contract_block_explorer.md
+
+
+### Overview
+
+Hybrid takes a unique, partially decentralized approach that improves two major problems with current open source Substrate block explorers: centralization and huge hosting requirements.
+
+A fully centralized block explorer typically populates an SQL database with the entirety of an archive node and stores additional data to index everything. Operating such a database reliably requires huge system resources and expense.
+
+When querying block information, or the chain state at any block height, the Hybrid dapp will use the [Substrate Connect](https://substrate.io/developers/substrate-connect/) light client from within the browser. Alternatively, these queries can be made directly to an archive node via WSS.
+
+For event search functionality, the Hybrid indexer efficiently indexes events in all blocks so they can be found with a simple WSS query. For example, to find all events connected with a specific `AccountId`.
+
+This architecture has three main advantages:
+- state queries are fully decentralized - you don't have to trust an RPC provider not to lie to you
+- 100% availability - the light client doesn't depend on any centralized service that may not always be available
+- the Hybrid indexer has significantly lower system requirements - it doesn't need to store all chain history
+
+Eventually, Hybrid will use this centralized / decentralized approach as the basis for an ink! contract explorer.
+
+Because Substrate is a federated platform, it will be possible browse multiple chains from the Hybrid dapp.
+
+### Project Details
+
+Hybrid will be a Substrate block explorer dapp. By default it will connect to major Substrate blockchains. Additionally it can be configured to connect to any Substrate chain.
+
+There are two types of queries that the explorer can perform:
+- Decentralized - ideally using the Substrate Connect light client, or alternatively connecting to an archival node via WSS. For example, querying blocks or state storage.
+- Centralized - the Hybrid indexer for the chain will be queried via WSS. This is for event searching and watching, such as finding all events across all chains that relate to a specific `AccountId`.
+
+![Hybrid Architecture](https://raw.githubusercontent.com/ethernomad/hybrid-diagram/main/hybrid.png)
```

Comment:
Since the dapp is shows as `Decentralised` here: have you looked into hosting the frontend via Crust/IPFS/...?

Reply:
I just updated the Future Plans section to mention IPFS hosting of the frontend.